src/Simulator/IncrementalRedstoneSimulator.cpp

```diff
@@ -1160,17 +1160,33 @@ void cIncrementalRedstoneSimulator::HandlePressurePlate(int a_RelBlockX, int a_R
 		{
 			// MCS feature - stone pressure plates can only be triggered by players :D
 			cPlayer * a_Player = this->m_World.FindClosestPlayer(Vector3f(BlockX + 0.5f, static_cast<float>(a_RelBlockY), BlockZ + 0.5f), 0.5f, false);
+			static Int64 m_PlayerLastOnPlate = 0;
```

Comment:
Exactly. This PR is a mess and merging it was a complete mistake. We;ve got some trigger-happy bears in our team :P